Source/Core/Trackers/InspirationTracker.cs

```diff
@@ -47,53 +51,84 @@ public static float GetInsiprationValue(Pawn pawn, bool refresh = false)
 
         public static void Calculate(Map map)
```

Comment:
Nice one. Only one cosmetic issue. Could you wrap those one line blocks with braces? Beside its good to merge.